docs: --verbose option in dev mode is not yet supported (#79366)
vercel/next.js: docs/01-app/02-guides/local-development.mdx

### Detailed fetch logging

> Use this flag in your `next.config.js` file, to see more detailed information about what's happening during development:

Author: ```suggestion
Use the `logging.fetches` option in your `next.config.js` file, to see more detailed information about what's happening during development:
```
